src/legacy/core_plugins/data/public/search/search_bar/components/saved_query_management/saved_query_management_component.tsx

```diff
@@ -63,40 +63,23 @@ export const SavedQueryManagementComponent: FunctionComponent<Props> = ({
   const [count, setTotalCount] = useState(0);
   const [activePage, setActivePage] = useState(0);
 
-  const fetchSavedQueries = async () => {
-    const savedQueryItems = await savedQueryService.findSavedQueries(
-      undefined,
-      perPage,
-      activePage + 1
-    );
-    const sortedSavedQueryItems = sortBy(savedQueryItems, 'attributes.title');
-    setSavedQueries(sortedSavedQueryItems);
-  };
-
-  const fetchSavedQueryCount = async () => {
-    const savedQueryCount = await savedQueryService.getSavedQueryCount();
-    setTotalCount(savedQueryCount);
-  };
-
   useEffect(() => {
-    // Only fetch the total count on initial render.
     const fetchCountAndSavedQueries = async () => {
-      await fetchSavedQueryCount();
-      await fetchSavedQueries();
+      const savedQueryCount = await savedQueryService.getSavedQueryCount();
```

Comment:
The savedQueryService isn't available in the component until it mounts, that's why I had to move these calls to the service back inside the effect that only runs once the component is open and mounted.

Because we have a large number of saved queries per page, page changes probably won't happen all that often, so I don't think that calling for the count every time the page changes is too much to be concerned over.